slack_messages.gs

```diff
@@ -101,6 +106,140 @@ function volunteerSuccessReply(slackurl, uniqueid, requesterName, mention_reques
   });
 }
 
+var cancelSuccessMessage = function(row){
+  return textToJsonBlocks(`You just cancelled your offer for help for ${requestFormatted(row)}.
+      I've notified the channel.`);
```

Comment:
I think anything in backtick-quotes is formatted literally, so the line break and tabs will be passed on to the message.